blog-pipeline/tests/test_preflight_smoke_new.py

```
@@ -0,0 +1,399 @@
+        for name in ["PYTHONPATH completeness", "venv_python executable",
+                      "Output dir writable", "Step 5 write path",
+                      "Step function imports", "Config path consistency"]:
+            # Find the CheckResult line for this check
+            idx = source.index(f'"{name}"')
+            snippet = source[idx:idx+100]
+            assert '"critical"' in snippet, f"Check '{name}' should be critical, got: {snippet}"
+
+        # Warning check
+        idx = source.index('"DataForSEO balance"')
+        snippet = source[idx:idx+100]
+        assert '"warning"' in snippet, f"DataForSEO balance should be warning severity"
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)